src/fastga_he/models/loops/wing_area_component/wing_area_cl_dep_equilibrium.py

```diff
@@ -69,6 +69,12 @@ def initialize(self):
             desc="Boolean to sort the component with proper order for adding subsystem operations",
             allow_none=False,
         )
+        self.options.declare(
+            name="low_speed_aero",
+            default=False,
+            desc="Boolean to consider low speed aerodynamics",
+            allow_none=False,
+        )
```

Comment:
Actually, this shouldn't be an option. For the computation of the low speed equilibrium we will *ALWAYS* use low speed ! The only place this option should appear in this file is on line 447.

This means the test you added should also be removed and the group should also not have the option